tests/amr/data/particles/refine/test_tag_strategy.h

```diff
@@ -131,19 +149,22 @@ class TagStrategy : public SAMRAI::mesh::StandardTagAndInitStrategy
 
                     auto& interior = particlesData->domainParticles;
 
-                    auto particlesBox      = particlesData->getBox();
-                    auto particlesGhostBox = particlesData->getGhostBox();
-                    // particlesBox           = AMRToLocal(
-                    //    static_cast<std::add_const_t<decltype(particlesBox)>>(particlesBox),
-                    //    particlesGhostBox);
+                    auto particlesBox = particlesData->getBox();
 
-                    // here we are 1D
-                    for (int iCellPos = particlesBox.lower(dirX);
-                         iCellPos <= particlesBox.upper(dirX); ++iCellPos)
+                    auto lower = boxBoundsLower<dimension>(particlesBox);
+                    auto upper = boxBoundsUpper<dimension>(particlesBox);
+
+                    for (auto iCellZ = lower[dirZ]; iCellZ <= upper[dirZ]; ++iCellZ)
```

Comment:
for consistency in readibility with the rest of the code (even if not algorithmically important here) it could be for x, for y, for z

Reply:
if it can be it probably should be I suppose